.changeset/shaky-beans-reply.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/core': patch
+---
+
+Improved `agent.generate()` and `agent.stream()` performance by avoiding unnecessary workflow storage access.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)